sdk/python/feast/diff/FcoDiff.py

```diff
@@ -16,23 +16,28 @@
 from feast.protos.feast.core.RequestFeatureView_pb2 import (
     RequestFeatureView as RequestFeatureViewProto,
 )
+from feast.registry import FeastObjectType, Registry
+from feast.repo_contents import RepoContents
 
-FcoProto = TypeVar(
-    "FcoProto",
-    EntityProto,
-    FeatureViewProto,
-    FeatureServiceProto,
-    OnDemandFeatureViewProto,
-    RequestFeatureViewProto,
-)
+FEAST_OBJECT_TYPE_TO_STR = {
```

Comment:
non-blocking, but a) probably could have this in the enum or b) this map should live closer to the plan logic